### PR TITLE
closes #3 build default class schedule ui

### LINKED ISSUE #3: Class details page
[בניית מערכת שעות דיפולטית ברמת כיתה.docx](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5ae80ca603d1cb001202249b/125516c66e82c728ace21e0d465f652de3357a1e765e750313a347569e8e90889ecee39cb932219f4af5b141b8fca905a6ccb8e11b556e83411fdbea9abbaa0a498af1935c2df6075f2bf825e61e32975dc5f4e3d01993ac2f2ff58edd4f3b4a004980ad5c35f1914fc858f719e21e54a3406471fa64742d19c0412875213264b9f6cd578e8f61a1b8182f5fc1e5e6fa1f12a86e95.docx)
[שיעורים וסמלים.pptx](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5ae80ca603d1cb001202249b/125516c66e82c728ace21e0d465f5d2dda35411e7a5e77f45dea0dd49eb390b29efa14d7c77c54d5fd0e39111d0014f9121c4d4eeefcdb2ba3e23921830e0aed80655b74c63c43afeb8318ec53a83c990dd4e8e8d51199a26867f5819d4e3614041d8bf75d3ca99f4fc850f14cf300.pptx)
[מקומות.pptx](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5ae80ca603d1cb001202249b/125516c66e82c728ace21e0d465f6a2de435761e715e4af46e1fbcd6395428792c07b7692ac7b62aef177f540e561cad41095740fcf98879a5e53d77d90e19f1c32f0e2a92624cf5f2d75dad0ba821824584b3be824dc0ed3b22e28b884d2c53.pptx)

parent of #90
parent of #91
parent of #92
parent of #94
parent of #96
parent of #97
parent of #207
need #90
need #91
need #92
need #94
need #96
need #97
need #207

### PR DESCRIPTION
**Pull request by itays**

### CODE REVIEW

File: src/app/components/schedule/schedule-dialog/schedule-dialog-data.model.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { TimeSlot } from '../../../models/timeslot.model';
+
+export interface ScheduleDialogData extends TimeSlot {
```
Comment (eran1205) [inline src/app/components/schedule/schedule-dialog/schedule-dialog-data.model.ts]: Naming convention? Is this how we want to declare as an interface?

File: src/app/pages/class-details/class-details.container/class-details.container.component.ts [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+import { MatDialog } from '@angular/material';
+import { ScheduleDialogComponent } from '../../../components/schedule/schedule-dialog/schedule.dialog';
+import { ScheduleDialogData } from '../../../components/schedule/schedule-dialog/schedule-dialog-data.model';
+import { Class } from '../../../models/class.model';
+import { ClassDetailsEventParams } from '../class-details.view/class-details.view.component';
+import { ScheduleService } from '../../../services/schedule/schedule.service';
+
+@Component({
+  selector: 'app-class-details-container',
+  template: `<app-class-details-view
```
Comment (eran1205) [inline src/app/pages/class-details/class-details.container/class-details.container.component.ts]: Don't we want to use templateUrl?

File: src/app/pages/class-details/class-details.view/class-details.view.component.scss [hunk 3]
```diff
@@ -0,0 +1,14 @@
+.class-details-container {
+    direction: rtl;
+    text-align: right;
+    h2 {
+        font-size: 32px;
+    }
+}
+.class-detail-field {
+    display: inline-block;
+    font-family: Rubik;
```
Comment (eran1205) [inline src/app/pages/class-details/class-details.view/class-details.view.component.scss]: new font-family? why do we need to define it here?

File: src/app/pages/class/services/class.graphql.service.ts [hunk 4]
```diff
@@ -2,79 +2,165 @@ import { Injectable } from '@angular/core';
...
+              _id
+              level
+              name
+            }
+          }
+        `,
+      })
+      .toPromise();
+  }
+  classById(id: string) {
```
Comment (eran1205) [inline src/app/pages/class/services/class.graphql.service.ts]: Can we make one object: classById and classbyName are the same...

File: src/app/pages/student/dialogs/update/services/student.graphql.service.ts [hunk 5]
```diff
@@ -0,0 +1,91 @@
+import { Injectable } from '@angular/core';
```
Comment (eran1205) [inline src/app/pages/student/dialogs/update/services/student.graphql.service.ts]: Delete this file. it's redundent

File: src/app/services/schedule/schedule.service.ts [hunk 6]
```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@angular/core';
+
+@Injectable()
+export class ScheduleService {
+  levels = ['א', 'ב', 'ג', 'ד', 'ה', 'ו'];
```
Comment (eran1205) [inline src/app/services/schedule/schedule.service.ts]: I think that we need to define it as a separate ENUM for reusable

File: src/assets/scss/dialogs.scss [hunk 7]
```diff
@@ -0,0 +1,11 @@
+.msw-dialog {
+    direction: rtl;
+    text-align: right;
+    padding: 0;
+}
+.mat-dialog-title {
+    font: 600 20px Rubik;
```
Comment (eran1205) [inline src/assets/scss/dialogs.scss]: font-family?

File: src/app/pages/class-details/class-details.container/class-details.container.component.ts (hunk 2 above)
Comment (itays) [reply src/app/pages/class-details/class-details.container/class-details.container.component.ts]: seems redundant to create a another html file and request it for just a few lines of html

File: src/mocks/assets/schedule.mock.ts [hunk 8]
```diff
@@ -0,0 +1,298 @@
+export const scheduleTestData = [
```
Comment (dimkinv) [inline src/mocks/assets/schedule.mock.ts]: Why do we need this file?

Comment (itays) [reply src/mocks/assets/schedule.mock.ts]: used it as a mock. theoretically we don't need it anymore

**Pull request merged**